### PR TITLE
[cordova] update pack scripts for beta configuration

### PR DESCRIPTION
**Pull request by zhuyongyong**
- Crosswalk beta configuration in Webview plugin need update in packaged scripts
- For canary crosswalk app, install crosswalk in script
- Build a variety of cordova app successfully 

BUG=https://crosswalk-project.org/jira/browse/XWALK-4903

### CODE REVIEW

**Comment by Honry** [comment]
Please also update the script in all-in-one suite.

File: misc/webapi-noneservice-tests/pack.sh [hunk 1]
```diff
@@ -152,7 +159,11 @@ elif [ $pack_type == "cordova" ]; then
         for plugin in `ls $BUILD_ROOT/cordova_plugins`
         do
             if [ $plugin == "cordova-plugin-crosswalk-webview" ]; then
-                cordova plugin add $BUILD_ROOT/cordova_plugins/$plugin --variable XWALK_VERSION="$crosswalk_version" --variable XWALK_MODE="$pack_mode"
+                if [ $crosswalk_branch == "beta" ]; then
+                    cordova plugin add $BUILD_ROOT/cordova_plugins/$plugin --variable CROSSWALK_ANDROID_VERSION="org.xwalk:xwalk_core_library_beta:$crosswalk_version" --variable LIB_MODE="$pack_mode"
+                else
```
Comment (Honry) [inline misc/webapi-noneservice-tests/pack.sh]: What about canary build in all-in-one suite?

**Comment by Honry** [comment]
@qiuzhong, please help to verify this PR in your weekly release environment.

**Comment by qiuzhong** [comment]
Verified. It's OK for beta branch to pack without needs to modify platform crosswalk webview plugin.

**Comment by Honry** [comment]
Thanks @qiuzhong!
@zhuyongyong, please also cherry-pick to Crosswalk 15 branch.

**Pull request merged**